### ISSUE TITLE
SDL Asserts

### ISSUE THREAD

**Issue by mrwonko** [JACoders/OpenJK#715]
[SDL has some clever assertion code](https://plus.google.com/+RyanGordon/posts/KzV6sLdALX7); we should probably use that outside the dedicated code.

**Comment by Razish** [comment]
Definitely, I was about to suggest this =P

**Comment by mrwonko** [comment]
Although this would not work in the libraries, just the client, since only that is linked to SDL?

**Comment by mrwonko** [comment]
I believe SDLTest is actually a separate static library, so we could conceivably link to that everywhere.

**Comment by ensiform** [comment]
Unfortunately SDL is not used everywhere.  It's not included at all in the dedicated server.

**Comment by mrwonko** [comment]
Yeah, but I'm saying it could be - the assert sublibrary at least.

**Comment by ensiform** [comment]
No because we don't want to force servers to have to require SDL at all.

**Comment by mrwonko** [comment]
It's just a compile time dependency though, a static library. No dll required.
